### PR TITLE
Add PHP Code Sniffer

### PR DESCRIPTION
**Pull request by msilvagarcia**

### CODE REVIEW

File: adyen.php [hunk 1]
```diff
@@ -20,6 +20,8 @@
  * See the LICENSE file for more info.
  */
 
+// phpcs:disable PSR1.Classes.ClassDeclaration
+
```
Comment (rikterbeek) [inline adyen.php]: What does this do exactly ?

File: controllers/admin/AdminAdyenPrestashopCronController.php [hunk 2]
```diff
@@ -13,13 +13,15 @@
  *                               #############
  *                               ############
  *
- * Adyen PrestaShop module
+ * Adyen PrestaShop plugin
```
Comment (rikterbeek) [inline controllers/admin/AdminAdyenPrestashopCronController.php]: Prestashop uses the temonology modules https://addons.prestashop.com/en/ shall we call it a module ? ask Grant to verify this.

File: controllers/front/Notifications.php [hunk 3]
```diff
@@ -20,6 +20,8 @@
  * See the LICENSE file for more info.
  */
 
+// phpcs:disable PSR1.Classes.ClassDeclaration
+
```
Comment (rikterbeek) [inline controllers/front/Notifications.php]: same here ?

File: adyen.php (hunk 1 above)
Comment (msilvagarcia) [reply adyen.php]: Disables warning about a class not being part of a namespace.

**Pull request merged**